### PR TITLE
Task 1 - Maven project with basic TestNG and Selenium configuration

### PR DESCRIPTION
**Pull request by odibranin**

### CODE REVIEW

File: src/test/resources/testing.xml
Comment (nejlamaksumic-abh) [inline src/test/resources/testing.xml:1]: file should be named `testng.xml`

File: src/test/java/com/odibranin/qamp/placelab/tests/SmokeTest.java [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+import com.odibranin.qamp.placelab.utils.WebDriverSetup;
+import org.openqa.selenium.WebDriver;
+import org.testng.annotations.AfterSuite;
+import org.testng.annotations.BeforeSuite;
+import org.testng.annotations.Test;
+
+public class SmokeTest {
+    private WebDriver driver;
+
+    @BeforeSuite
```
Comment (nejlamaksumic-abh) [inline src/test/java/com/odibranin/qamp/placelab/tests/SmokeTest.java:12]: why `BeforeSuite` and not `BeforeTest`?

File: pom.xml [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+            <groupId>io.github.bonigarcia</groupId>
+            <artifactId>webdrivermanager</artifactId>
+            <version>${webdrivermanager.version}</version>
+        </dependency>
+
+        <dependency>
+            <groupId>org.testng</groupId>
+            <artifactId>testng</artifactId>
+            <version>${testng.version}</version>
+            <scope>compile</scope>
```
Comment (nejlamaksumic-abh) [inline pom.xml:37]: why scope compile?

File: pom.xml [hunk 3]
```diff
@@ -0,0 +1,58 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <groupId>org.example</groupId>
+    <artifactId>AutomationHomework1</artifactId>
```
Comment (nejlamaksumic-abh) [inline pom.xml:8]: with artifactId being the unique name for your project, consider it changed to placelab-ui-tests or something similar, consider updating the group Id as well

Comment (odibranin) [reply pom.xml:8]: Fixed.

File: src/test/resources/testing.xml
Comment (odibranin) [reply src/test/resources/testing.xml:1]: Fixed.

File: src/test/java/com/odibranin/qamp/placelab/tests/SmokeTest.java (hunk 1 above)
Comment (odibranin) [reply src/test/java/com/odibranin/qamp/placelab/tests/SmokeTest.java:12]: Fixed.

File: pom.xml (hunk 2 above)
Comment (odibranin) [reply pom.xml:37]: Fixed.

**Pull request merged**